queries/injections.scm

```diff
@@ -1,9 +0,0 @@
-((macro_invocation
-  (token_tree) @injection.content)
```

Comment:
This had to get removed because a `token_tree` is no longer a valid grammar node I think

This is basically pointing at the fact that this "version"/fork of the grammar wouldn't be used in a context where you could/would be doing injection in order to re-parse macro invocation contents